### PR TITLE
Add caption for jmedinacsu

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @rogeryanda
- [ ] @jmedinacsu
- [ ] @js3263827
- [ ] @kayakkat
- [ ] @rjenkins5725
- [ ] @liam09
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @IAmHughes
- [ ] @edelgadojr
- [ ] @Phat6
- [ ] @octowilliam
- [ ] @jrhixson2002
- [ ] @dannyreid
- [ ] @mgthompson2811
- [ ] @anshubhatia
- [ ] @rogeryanda
- [ ] @githubteacher
- [ ] @Phat6
- [ ] @dannyreid
- [ ] @jrhixson2002
- [ ] @js3263827
- [ ] @jmedinacsu
- [ ] @IAmHughes
- [ ] @liam09
- [ ] @rjenkins5725
- [ ] @edelgadojr
- [ ] @rogeryanda
- [ ] @octowilliam
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @rtsinclair
- [ ] @rogeryanda
- [ ] @gmeyer39
- [ ] @rtsinclair
- [ ] @lkelly1
- [ ] @js3263827
- [ ] @githubteacher
- [ ] @githubteacher
- [ ] @kayakkat
- [ ] @jrhixson2002
- [ ] @jrhixson2002
- [ ] @gmeyer39
- [ ] @lgibbens

### PR DESCRIPTION
**Pull request by jmedinacsu**
TL;DR - One sentence summary of whats going on.

## Tasks remaining
- [ ] Get a review

Link to issue of service now
This resolves #2 

Leading questions

### CODE REVIEW

File: _posts/0000-01-03-jmedinacsu.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is my new caption...
```
Comment (dannyreid) [inline _posts/0000-01-03-jmedinacsu.md:6]: Are you sure its new :fried_shrimp:

Comment (dannyreid) [reply _posts/0000-01-03-jmedinacsu.md:6]: Wurd :word:

**Review by dannyreid — COMMENTED** [review]
Push it to prod
@jmedinacsu

**Review by dannyreid — CHANGES_REQUESTED** [review]
I need to leave a comment

**Review by jmedinacsu — COMMENTED** [review]
requested changes made.

File: _posts/0000-01-03-jmedinacsu.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
...
```
Comment (dannyreid) [inline _posts/0000-01-03-jmedinacsu.md:6]: We are done here :shark:

**Review by dannyreid — APPROVED** [review]
